src/features/ballot/components/BallotOverview.tsx

```diff
@@ -108,11 +106,15 @@ function BallotOverview() {
         </Button>
       ) : canSubmit ? (
         <SubmitBallotButton disabled={sum > config.votingMaxTotal} />
-      ) : allocations.length ? (
+      ) : isRegistered && isAllowedToVote && allocations.length ? (
```

Comment:
```suggestion
      ) : isRegistered && allocations.length ? (
```
if `isRegistered` is true, then `isAllowedToVote` should always be true, right?